controllers/profiles/common/object_creators.go

```diff
@@ -76,6 +76,7 @@ const (
 // It serves as a basis for a basic Quarkus Java application, expected to listen on http 8080.
 func DeploymentCreator(workflow *operatorapi.SonataFlow, plf *operatorapi.SonataFlowPlatform) (client.Object, error) {
 	lbl := workflowproj.GetMergedLabels(workflow)
+	delete(lbl, workflowproj.LabelK8SPartOF)
```

Comment:
I'll do that and in addition I think it is the best to also use the platform object we have there and not use the platform from the status of the workflow because it is not there - hence the errors in the controller log while trying to update the deployment selector
wdyt?